_data/landscape/meshes.yml

```diff
@@ -25,6 +28,8 @@
   tracing: "Yes"
   encryption: "Yes"
   multi-tenant-score: 6
+  spiffe: "?"
+  spire: "?"
```

Comment:
Not supported today, but in development https://github.com/aws/aws-app-mesh-roadmap/issues/68